habitat-lab/habitat/config/benchmark/rearrange/cat_gaze.yaml

```diff
@@ -0,0 +1,56 @@
+# @package _global_
+
+defaults:
+  - /habitat: habitat_config_base
+  - /habitat/simulator/agents@habitat.simulator.agents.main_agent: depthp_head_rgb_third_agent
+  - /habitat/task/rearrange: cat_gaze
+  - /habitat/dataset/rearrangement: floorplanner
+  - _self_
+
+habitat:
+  gym:
+    auto_name: Pick
+    obs_keys:
+      - robot_head_rgb
+      - robot_head_depth
+      - obj_start_sensor
+      - object_embedding
+      - joint
+      - is_holding
+      - relative_resting_position
+  environment:
+    max_episode_steps: 300
+  simulator:
+    type: ObjectRearrangeSim-v0
+    additional_object_paths:
+      - data/objects/ycb/configs/
+      - data/objects/amazon_berkeley/configs/
+      - data/objects/google_object_dataset/configs/
+    debug_render_goal: False
+    debug_render: False
+    needs_markers: False
+    concur_render: True
+    auto_sleep: True
+    requires_textures: False
+    kinematic_mode: True
+    agents:
+      main_agent:
+        radius: 0.25
+        robot_urdf: data/robots/hab_stretch/urdf/hab_stretch.urdf
+        robot_type: "StretchRobot"
+        sim_sensors:
+           head_depth_sensor:
+             height: 256 # 640
```

Comment:
Can we remove the commented values?

Reply:
Just wanted to keep the real robot specs. This was added to all configs.

Reply:
I think we'll have a separate config for evaluation anyway, right? We can use the real robot specs there and remove these comments.

Reply:
You're right. We no longer plan to train with high resolution. We may try this later if the performance is severely low.